README.md

```diff
@@ -215,7 +232,8 @@ See the [GitHub Releases page](https://github.com/twbs/bootlint/releases) for de
 * 2014-10-16 - v0.5.0: Adds several new features. Adds official bookmarklet. Disables auto-lint-on-load in browser. Tweaks some checks. **Not backward compatible**
 * 2014-10-07 - v0.4.0: Adds checks for correct Glyphicon usage and correct modal DOM structure; fixes `.panel-footer` false positive
 * 2014-09-26 - v0.3.0: Several bug fixes and enhancements. **Not backward compatible**
-* 2014-09-23 - v0.2.0: First formal release. Announcement: <http://blog.getbootstrap.com/2014/09/23/bootlint/>
+* 2014-09-23 - v0.2.0: First formal release. Announcement: https://blog.getbootstrap.com/2014/09/23/bootlint/
```

Comment:
Because, we might use CloudFlare, but in general we don't use the https URLs for our links. Also, the angle brackets aren't cosmetic and should be there.